news/2020-05-27-new-featured-artist-thank-you-scientist.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "New Featured Artist: Thank You Scientist"
+date: 2020-05-27 08:00:00 +0000
+---
+
+**Thank You Scientist** adds some method to the madness as our latest addition to the Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/83/header.jpg)
+
+Our quest to encompass just about every genre and musical style continues this week with our latest snag, the seven-man wonderband that is [**Thank You Scientist**](https://osu.ppy.sh/beatmaps/artists/83). 
+
+Turning heads since 2012, the New Jersey based septet is widely renowned for their ecletic sound drawing upon influences from dozens of genres and folding them all under the progressive banner.
```

Comment:
```suggestion
Turning heads since 2012, the New Jersey-based septet is widely renowned for their eclectic sound that draws upon influences from dozens of genres and folds them all under the progressive banner.
```